Source/WebCore/page/Chrome.cpp

```diff
@@ -353,9 +352,9 @@ void Chrome::mouseDidMoveOverElement(const HitTestResult& result, unsigned modif
     String toolTip;
     TextDirection toolTipDirection;
     getToolTip(result, toolTip, toolTipDirection);
-    m_client->mouseDidMoveOverElement(result, modifierFlags, toolTip, toolTipDirection);
+    m_client->mouseDidMoveOverElement(result, modifiers, toolTip, toolTipDirection);
 
-    InspectorInstrumentation::mouseDidMoveOverElement(m_page, result, modifierFlags);
+    InspectorInstrumentation::mouseDidMoveOverElement(m_page, result, modifiers.toRaw());
```

Comment:
As discussed offline, we should try to get rid of the `toRaw` here if possible.

Reply:
Fixed in https://github.com/WebKit/WebKit/commit/1b59364ae200544534b097906f0eca943e445b7a